File: EJERCICIOS EN CLASE/CONDICION_estudiantes/Condicion_estudiantes.java
import java.util.Scanner;

public class Condicion_estudiantes{

    
    private static final float NOTA_APROBACION = 7.0f;
    private static final int TOTAL_NOTAS = 3;

    public static void main(String[] args) {
        try (var scanner = new Scanner(System.in)) {
            float[] notas = new float[TOTAL_NOTAS];
            float suma = 0;

       
        for (int i = 0; i < TOTAL_NOTAS; i++) {
            System.out.printf("Ingrese la nota %d: ", i + 1);
            notas[i] = scanner.nextFloat();
            suma += notas[i];
        }

        float promedio = suma / TOTAL_NOTAS;

        
        System.out.println("\n--- RESUMEN DE NOTAS ---");
        for (int i = 0; i < TOTAL_NOTAS; i++) {
            System.out.printf("Nota %d: %.2f%n", i + 1, notas[i]);
        }

        System.out.printf("Promedio final: %.2f%n", promedio);

        String estado = (promedio >= NOTA_APROBACION) ? "Aprobado" : "Reprobado";
        System.out.println("Estado del estudiante: " + estado);

        }
    }
}

File: EJERCICIOS EN CLASE/CONDICION_estudiantes/Condicion_estudiantes.cpp
#include <iostream>
using namespace std;

int main() {
    float nt1, nt2, nt3, promedio;

    cout << "Ingrese la nota 1: ";
    cin >> nt1;

    cout << "Ingrese la nota 2: ";
    cin >> nt2;

    cout << "Ingrese la nota 3: ";
    cin >> nt3;

    promedio = (nt1 + nt2 + nt3) / 3;

    cout << "Su primera nota es: " << nt1 << endl;
    cout << "Su segunda nota es: " << nt2 << endl;
    cout << "Su tercera nota es: " << nt3 << endl;
    cout << "Su promedio es: " << promedio << endl;

    if (promedio >= 7) {
        cout << "Está aprobado" << endl;
    } else {
        cout << "Está reprobado" << endl;
    }

    return 0;
}